R/cieplot.R

```diff
@@ -53,11 +52,12 @@
 
 cieplot <- function(ciedata, mono = TRUE, out.lwd = NULL, out.lcol = "black",
                     out.lty = 1, theta = 45, phi = 10, r = 1e6, zoom = 1, box = FALSE,
-                    margin = c(0, 0, 0, 0), ciebg = TRUE, ...) {
-
-  oldpar <- par(no.readonly = TRUE)
-  on.exit(par(oldpar))
+                    ciebg = TRUE, margin = NULL, ...) {
 
+  if (!missing("margin"))
+    message("The 'margin' argument is deprecated, and will be ignored. See ?par() for guidance on 
+            setting margins in the standard manner.")
```

Comment:
The alignment might not be what you expect here. Can you double check if that's what you want? Below is a proposal of what I think you meant:

```suggestion
    message(
      "The 'margin' argument is deprecated, and will be ignored. ", 
      "See ?par() for guidance on setting margins in the standard manner."
    )
```

On a related note, shouldn't this be a warning?